docs/examples/rs-tutorial.rst

```diff
@@ -158,9 +153,53 @@ Train-Test-Compare
 This pipeline gives a more complicated example of how to create visualizations that may be helpful for understanding the effectiveness of a model. The **EvalCompare** node provides a simple comparison visualization of two models.
 
 
+Train-PredVis
+~~~~~~~~~~~~~
+.. figure:: vis-pred.png
+    :align: center
+
+This pipeline shows another more complex and useful visualization example that can be helpful for understanding the effectiveness of your redshift estimation model. It generates a set of graphs like the one below that show the output probability distribution function (pdf) for the redshift values of a set of random galaxies' images. A pair of vertical  lines in each subplot indicate the actual redshift value (green) and the predicted redshift value (red) for that galaxy.
+
+As shown in this example, any visualization that can be created using the `matplotlib.pyplot <https://matplotlib.org/3.2.2/api/_as_gen/matplotlib.pyplot.html>`_ python library can be created and displayed by a pipeline. Displaying these visualizations can be accomplished by calling the **pyplot.show()** function after building the visualization. They can then be viewed from the `Executions view <../fundamentals/interface.rst#Executions>`_.
```

Comment:
I suspect that this may not be strictly true, but I am not fully aware of the limitations. Are there known features in pyplot that cannot be used? Also, are there other visualization libraries that can be used instead of pyplot? In either case, this paragraph may need to be changed slightly to reflect that.

Reply:
I think it is fine. We support most of the standard plots and will support more after #1747. Even after #1747, there are a few we still won't support such as pie charts but I don't think that detail is necessary. If you wanted to say "most" that might be a bit more accurate but it isn't a big deal.

Reply:
I will keep it as is in that case. I suspect that if we had a large number of users trying to use an unsupported plot, it would be best to consider either implementing support for explicitly mentioning that they are unsupported with no plans for support, but I would not include it now as it may discourage feedback about what types of missing plotting functionality would be helpful.